### ISSUE TITLE
Django 2.0

### ISSUE THREAD

**Issue by burgundy93** [GetBlimp/django-rest-framework-jwt#433]
Any plans for Django 2.0 support?

**Comment by kamilgregorczyk** [comment]
It seems to work for me, do you have any problems?

**Comment by gyukebox** [comment]
If it is compatible with django 2.0, wouldn't be better if the docs could be updated? - #435

**Comment by burgundy93** [comment]
I got it working. :D Thanks guys!

**Comment by budiardy** [comment]
It doesn't work well with django 2.0. 
The issue is: failed to overwrite 'JWT_RESPONSE_PAYLOAD_HANDLER'  setting.
the default is:
'JWT_RESPONSE_PAYLOAD_HANDLER': 'rest_framework_jwt.utils.jwt_response_payload_handler',
want to become:
'JWT_RESPONSE_PAYLOAD_HANDLER': 'jwt_response_payload_handler',
 i put the "def jwt_response_payload_handler" in the same page in the django settings file.
I guess it because the different version of Django

**Comment by jadhavmanoj** [comment]
Any update on support django 2.x? I am now evaluating which JWT auth package use for python 3.x & Django 2.x.

**Comment by krdeepak** [comment]
Same here. I tried https://github.com/davesque/django-rest-framework-simplejwt but this seems to be more widely adopted. Is someone working on django 2.0 official support?

**Comment by jadhavmanoj** [comment]
@krdeepak I am used in Django 2.x. I don't see any issue.

**Comment by stunaz** [comment]
@jadhavmanoj but the tests are not against Django 2.x in this repo

**Comment by jadhavmanoj** [comment]
@stunaz true. currently, I am writing test cases for Django 2.1 support. I will send PR once I finished.

**Comment by jadhavmanoj** [comment]
@stunaz  Please check PR #479  All test cases executed except for python version 3.3. tox has some issue for python version 3.3.

**Comment by stunaz** [comment]
cool @jadhavmanoj . Now, good luck to get it merged. this project is abandoned, hasnt get updated for years ... see https://github.com/GetBlimp/django-rest-framework-jwt/issues/449
